### DISCUSSION TITLE
Calendar Event component

### DISCUSSION THREAD

**Discussion by vaseker** [zepp-health/zeppos-forum#154] [Q&A]
Hello!
Im just bought T-Rex Ultra as a replacement for AW5 and see no watchface with ability to show nearest event from my calendar.
Also I see no documentation related to registering new watchface component and API for callendar app messaging.

Is it possible to implement AW's "Your schedule" complication as Zepp watchface component?

Ideally, I want to port my "modular" AW's watchface configuration.

![photo_2023-04-27 18 20 16](https://user-images.githubusercontent.com/1942271/234874560-46633045-bfab-4eba-9781-8d34eeadd8dc.jpeg)
![photo_2023-04-27 18 20 13](https://user-images.githubusercontent.com/1942271/234874581-28ea5e88-ac63-4226-8792-35762db511c2.jpeg)

Thanks.

**Comment by zdunex25** [comment]
https://github.com/orgs/zepp-health/discussions/134
